generators/python/src/fern_python/generators/pydantic_model/fern_aware_pydantic_model.py

```diff
@@ -110,6 +115,15 @@ def to_reference(self) -> LocalClassReference:
     def get_class_name(self) -> str:
         return self._class_name
 
+    def _model_validator(self, pre: bool = False) -> AST.FunctionInvocation:
+        return AST.FunctionInvocation(
+            function_definition=AST.Reference(
+                qualified_name_excluding_import=("model_validator",),
+                import_=Pydantic.PydanticImport(),
+            ),
+            kwargs=[("mode", AST.Expression(expression='"before"' if pre else '"after"'))],
+        )
+
```

Comment:
I don't like having this as part of the class, but I also don't know where you store utility files in your repo. Feel free to move!